redisinsight/ui/src/pages/browser/modules/key-details/components/rejson-details/rejson-scalar/RejsonScalar.tsx

```diff
@@ -36,7 +36,15 @@ const RejsonScalar = (props: JSONScalarProps) => {
   const dispatch = useDispatch()
 
   useEffect(() => {
-    setChangedValue(isString(value) ? `"${value}"` : isNull(value) ? 'null' : value)
+    if (typeof value === 'bigint') {
+      setChangedValue(value.toString())
+    } else if (isString(value)) {
+      setChangedValue(`"${value}"`)
+    } else if (isNull(value)) {
+      setChangedValue('null')
+    } else {
+      setChangedValue(value)
+    }
```

Comment:
Do you mean like so -> https://github.com/RedisInsight/RedisInsight/pull/4089/commits/238acd98a5a502e1fb4ba3dae4c64554b96d2517 having a function wrapping up this functionality, instead of doing it directly in the useEffect, or something else?

Reply:
yes, but could you please move it to utils and call smth like stringifyScalarValue?

Reply:
done, covered it with tests as well https://github.com/RedisInsight/RedisInsight/pull/4089/commits/2d08a99edf1991533d0a28cc3516f3e81c473225

I usually wait until something needs the function from a different component,before separating it in a util, but I can see the benefit of the other approach as well